crates/cxx-qt-lib/src/core/qmetaobjectconnection.rs

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+use cxx::{type_id, ExternType};
+use std::mem::MaybeUninit;
+
+#[cxx::bridge]
+mod ffi {
+    #[namespace = "rust::cxxqtlib1"]
+    unsafe extern "C++" {
+        include!("cxx-qt-lib/qmetaobjectconnection.h");
+
+        #[doc(hidden)]
+        type QMetaObjectConnection = crate::QMetaObjectConnection;
+
+        #[doc(hidden)]
+        #[rust_name = "qmetaobjectconnection_disconnect"]
+        fn qmetaobjectconnectionDisconnect(conn: &QMetaObjectConnection);
+    }
+
+    #[namespace = "rust::cxxqtlib1"]
+    unsafe extern "C++" {
+        include!("cxx-qt-lib/common.h");
+
+        #[doc(hidden)]
+        #[rust_name = "qmetaobjectconnection_drop"]
+        fn drop(conn: &mut QMetaObjectConnection);
+    }
+}
+
+/// Represents a handle to a signal-slot (or signal-functor) connection.
```

Comment:
Please expand the documentation here a bit.

Especially the fact that a QMetaObjectConnection automatically disconnects when dropped!
As that's something we've changed compared to Qt.

Also, pointing out that you can connect to a signal using `on_SIGNAL_NAME` could quickly be explained here.

Reply:
Done, but maybe could explain better :thinking: